guy/my_ps/main.c

```diff
@@ -0,0 +1,53 @@
+#include <stdio.h>
+#include "proc_info.h"
+#include <stdbool.h>
+#include <dirent.h>
+
+/*
+Return 1 if the string contains only digits (0-9). 0 Otherwise.
+(Returns True for empty string)
+*/
+bool str_is_digit(const char *s) {
+    int i = 0;
+    while ('\0' != s[i]) {
```

Comment:
why while?

Reply:
A good question actually. (Changed to for)